client/transactions/list/style.scss

```diff
@@ -134,4 +134,64 @@ $gap-small: 12px;
 			height: auto;
 		}
 	}
+
+	.components-card__header {
```

Comment:
💭 It would be helpful to add a comment here explaining what the changes below achieve.

E.g. "These styles improve the overflow behaviour of the `Search` component with the `TableCard` when many tags are selected. Used for transaction list views. See PR #8996"

Reply:
Great point and helpful to explain the context in a comment. I have added it via 135a410